### DISCUSSION TITLE
v1.0.0-beta.1

### DISCUSSION THREAD

**Discussion by EvanMulawski** [EvanMulawski/FanControl.Razer#2] [Announcements]
Initial pre-release.

Supports the PWM Fan Controller (`0f3c`).

<hr /><em>This discussion was created from the release <a href='https://github.com/EvanMulawski/FanControl.Razer/releases/tag/v1.0.0-beta.1'>v1.0.0-beta.1</a>.</em>

**Comment by 325racer** [comment]
Just installed and so far it seems to be working well. I have to separate my two Radiator fans to separate hub ports (they are currently on a Y splitter and only one seems to be actively controlled).

But otherwise everything seems to be controlled properly.  Now to start building some curves. (This is a new PC, so have a lot to do/learn about what it needs)

**Comment by arlo-infinity** [comment]
@325racer Yeah, when you have two PWM fans on a Y splitter they see the exact same PWM signal and so they respond with the same % fan speed and you only see the RPM from one fan.  It's normal to need to use two ports for two fans.

**Reply by 325racer** [reply]
Understood, that's what I was expecting, however, only one fan seems to stop and be controlled on the 2 that are on a Y splitter. It's the 2 fans on the Radiator, which should run together, but I'll end up separating them and running both independently.

**Reply by arlo-infinity** [reply]
@325racer That's a little peculiar, the Y splitter might have faulty connections FYI in case you want to use it in future (or maybe that fan...).   But, yeah, if you have more than enough channels you can run them independently and have the freedom to put them under the same fan control parameters or if you want run them completely differently.  I think for the price, now with this Plugin and FanControl (Synapse isn't great), the Razer PWM Controller is the best way to go for pure PWM fan control.
